ui/app/models/role-ssh.js

```diff
@@ -120,6 +120,7 @@ export default Model.extend({
   }),
   algorithmSigner: attr('string', {
     helpText: 'When supplied, this value specifies a signing algorithm for the key',
+    possibleValues: ['', 'default', 'ssh-rsa', 'rsa-sha2-256', 'rsa-sha2-512'],
```

Comment:
Thanks for the swift change! 

Reviewing #14006 - since an empty string updates to `default` can we remove that from the array of `possibleValues` here? My apologies for the rigamarole, I noticed this while testing. It's slightly awkward UX to select nothing from the dropdown and then after saving have it _magically_ change to "Signing Algorithm: default"

 ## form

<img width="771" alt="Screen Shot 2022-11-11 at 1 49 23 PM" src="https://user-images.githubusercontent.com/68122737/201435657-ece3edd5-6625-41bc-bc0f-b1c1dfe7b873.png">

## config
<img width="505" alt="Screen Shot 2022-11-11 at 1 49 39 PM" src="https://user-images.githubusercontent.com/68122737/201435671-ac1f6381-8cbe-40d5-ba50-e638bff5285b.png">


@cipherboy -  not sure if you have an argument to keep it as a dropdown option? If so, then we can remove line 123 all together if the options are going to be the same as the backend as the dropdown complies automagically when 
 `AllowedValues` exist

Reply:
@hellobontempo that actually makes a lot of sense, made that change.